package.json

```diff
@@ -50,6 +50,7 @@
     "ember-code-snippet": "1.7.0",
     "ember-disable-prototype-extensions": "^1.1.0",
     "ember-export-application-global": "^1.0.5",
+    "ember-frost-popover": "^11.0.0",
```

Comment:
Shouldn't this be a _dependency_ and not a _devDependency_ since it is required by consumers for this codebase to function correctly?

Reply:
Which then means this needs to be a MAJOR release and not a PATCH